src/components/Widgets/Input/index.js

```diff
@@ -63,8 +62,6 @@ const InputContainer = ({
           className={className && `mr-2 ${className}`}
           value={value}
           onChange={onChange}
-          multiple={multiple && multiple}
-          size={multiple && "15"}
```

Comment:
Are you sure about removing it? I think it was added for a large drop-down by @Shruti3004

Reply:
I am unable to understand the meaning of `{multiple && multiple}` where `multiple` is a string according to `prop-type`.
Maybe @Shruti3004 could explain better.

Reply:
Updated it

Reply:
Yes it is for the multiple selection of files.